sensor/tests/resource/role/role_test.go

```diff
@@ -97,37 +99,39 @@ func matchBinding(namespace, id string) resource.MatchResource {
 }
 
 func (s *RoleDependencySuite) Test_BindingHasNoRoleId() {
-	s.testContext.RunBare("Binding should get an update if role gets created after", func(t *testing.T, testC *resource.TestContext, _ map[string]k8s.Object) {
-		deleteDep, err := testC.ApplyFileNoObject(context.Background(), "sensor-integration", NginxDeployment)
-		defer utils.IgnoreError(deleteDep)
-		require.NoError(t, err)
+	s.testContext.NewRun(
+		resource.WithTestCase(func(t *testing.T, testC *resource.TestContext, _ map[string]k8s.Object) {
+			deleteDep, err := testC.ApplyResourceNoObject(context.Background(), "sensor-integration", NginxDeployment, nil)
+			defer utils.IgnoreError(deleteDep)
+			require.NoError(t, err)
 
-		var binding v12.RoleBinding
-		deleteRoleBinding, err := testC.ApplyFile(context.Background(), "sensor-integration", NginxRoleBinding, &binding)
-		defer utils.IgnoreError(deleteRoleBinding)
-		require.NoError(t, err)
+			var binding v12.RoleBinding
```

Comment:
It's odd that github is showing this `tab` highlighted... Did you accidentally mixed spaces with tabs here or is it just a diff display issue?

Reply:
I think it's because before the changes the indentation was shifted to the left and now we have one more "level" of indentation to the right.